packages/plugin-graphql/src/index.js

```diff
@@ -65,10 +65,11 @@ class GraphQLResource extends ResourceInterface {
     return new Promise((resolve, reject) => {
       try {
         body = body.replace('<head>', `
+        
+        <head>
```

Comment:
Yeah, this looks good!  My only nitpick would be removing the extra line break but other wise this is desired approach for this fix.  👍